Sources/MapboxDirections/RouteOptions.swift

```diff
@@ -110,7 +110,7 @@ open class RouteOptions: DirectionsOptions {
     /**
      A number that influences whether the route should prefer or avoid alleys or narrow service roads between buildings.
      
-     This property has no effect unless the profile identifier is set to `DirectionsProfileIdentifier.walking`.
+     This property has no effect unless the profile identifier is set to `DirectionsProfileIdentifier.automobile` or `DirectionsProfileIdentifier.walking`.
```

Comment:
Maybe with this change it now make sense to set `alleyPriority` to `.low` by default if current profile is `.automobile`?
It alters current behavior, but maybe that is what we wanted with this change?

Reply:
I think the application or maybe the navigation SDK should be responsible for making that decision. As much as possible, MapboxDirections should align with the Directions API defaults. But the navigation SDK does vend a NavigationRouteOptions subclass that could set the `alleyPriority` to `.low` for `.automobile`.